arviz/utils.py

```diff
@@ -658,7 +658,7 @@ def _load_static_files():
 
     Clone from xarray.core.formatted_html_template.
     """
-    return [pkg_resources.resource_string("arviz", fname).decode("utf8") for fname in STATIC_FILES]
+    return [importlib.resources.read_text("arviz", fname) for fname in STATIC_FILES]
```

Comment:
```suggestion
    return [importlib.resources.files("arviz").joinpath(fname).read_bytes() for fname in STATIC_FILES]
```
I think it need to read bytes here